.github/workflows/ci.yml

```
@@ -57,6 +57,8 @@ jobs:
           poetry config http-basic.epc-power aws $CODEARTIFACT_AUTH_TOKEN
           poetry config repositories.third-party https://epc-power-us-${{ secrets.AWS_ACCOUNT_ID }}.d.codeartifact.us-east-2.amazonaws.com/pypi/third-party/
           poetry config http-basic.third-party aws $CODEARTIFACT_AUTH_TOKEN
+          # https://github.com/python-poetry/poetry/issues/7611
+          poetry config installer.max-workers 1
 
           poetry install
```

Missing the new poetry dynamic versioning plugin call.

```suggestion
          poetry self add "poetry-dynamic-versioning[plugin]"
          poetry install
```

---

[5f7d65e](https://github.com/epcpower/st/pull/41/commits/5f7d65e3b979978bbadfa8af1bc5e189748217a8)